include/aspect/simulator.h

```diff
@@ -883,21 +883,37 @@ namespace aspect
 
       /**
        * Fills a vector with the artificial viscosity for the temperature on
-       * each local cell
+       * each local cell.
        */
       void get_artificial_viscosity (Vector<float> &viscosity_per_cell) const;
 
       /**
        * Internal routine to compute the depth average of a certain quantitiy.
-       * The functor @p fctr should implement: 1. bool
-       * need_material_properties() 2. void setup(unsigned int q_points) 3.
-       * double operator()(const MaterialModelInputs & in, const
-       * MaterialModelOutputs & out, FEValues<dim> & fe_values, const
-       * LinearAlgebra::BlockVector &solution, std::vector<double> & output)
+       * The functor @p fctr should implement the following:
```

Comment:
I'm not quite sure what you're trying to say here. Is the intent to say something like
```
  The functor @p fctr must be an object of a user defined type that can be arbitrary but has to satisfy certain requirements. In essence, this class type needs to implement the following interface of member functions (possibly in addition to other member functions, but they will not be called by this current function):
```